tools/jdk/java_toolchain_alias.bzl

```diff
@@ -123,13 +86,11 @@ _java_runtime_transition = transition(
 )
 
 java_runtime_version_alias = rule(
-    implementation = _java_runtime_version_alias,
+    implementation = _java_runtime_alias,
```

Comment:
Maybe add some documentation here: IIUC this is now deprecated, and has the same behaviour as `_java_runtime_alias`, and anyone who wants to configure a specific Java major version should use a different approach?

Reply:
Just the implementation is the same. You missed `cfg = _java_runtime_transition` in the definition. `java_runtime_version_alias` still selects a specific Java major version. `java_runtime_alias` doesn't have the configuration transition.